specification/validated-relationships-service-api.yaml

```diff
@@ -1044,11 +1062,11 @@ components:
                                   enum: ["https://fhir.nhs.uk/CodeSystem/Proxy-Placeholder-ReasonForAccess"]                                
                                 code:
                                   type: string
-                                  enum: ['COMB', 'MHEA', 'PHYS', 'PRAC', 'TECH', 'NEUR', 'LRN', 'OTHR']
+                                  enum: ['COMB', 'MHEA', 'PHYS', 'PRAC', 'TECH', 'NEUR', 'LRN', 'PNS', 'OTHR']
```

Comment:
Exclusion of prefer not to say was intentional. If they've not said, then this optional question would be excluded from the response

Reply:
Have caught up with Rachel and we have decided to now include Prefer not to say for reasons for access, for now